app/views/public/index.html.erb

```diff
@@ -1,3 +1,16 @@
+<head>
+<script>  
+ $(window).scroll(function() {
+   var next_url = $("#infinite-scrolling a[rel='next']").attr('href');      
+   if (( next_url ) && ((window.innerHeight + window.pageYOffset) >= document.body.offsetHeight))
+   {          
+     $('#my-posts').show();
+     $.getScript(next_url);
+     return;
+   }
+ });
```

Comment:
can you move this js code into a separate js file?
keeping js code in html.erb is not a good practice

Reply:
Ok Magesh

Reply:
> can you move this js code into a separate js file? keeping js code in html.erb is not a good practice

It is showing this error when I am trying it in a seperate file Magesh
 
![image](https://user-images.githubusercontent.com/84360457/203964244-7bb0c9f3-93dd-42d3-a032-e7ba872053d3.png)

Reply:
Figure out a way to fix it